node/src/actors/chain_manager/mining.rs

```diff
@@ -227,6 +227,7 @@ impl ChainManager {
                     // This will run all the validations again
 
                     let block_hash = block.hash();
+                    // TODO. Currently last_block_proposed is not used, but removing it is a breaking change
```

Comment:
Maybe change the TODO as FIXME to track it and change it when there is a new release with breaking changes. :)